webapi/src/it/scala/org/knora/webapi/ITKnoraLiveSpec.scala

```diff
@@ -127,9 +127,7 @@ class ITKnoraLiveSpec(_system: ActorSystem) extends Core with StartupUtils with
         getResponseString(request).parseJson.asJsObject
     }
 
-
-
-    protected def singleAwaitingRequest(request: HttpRequest, duration: Duration = 5999.milliseconds): HttpResponse = {
+    protected def singleAwaitingRequest(request: HttpRequest, duration: Duration = 7999.milliseconds): HttpResponse = {
```

Comment:
Just in case you wonder why this odd number. It is easier to find the correct setting that needs to be tweaked. The timeout output is often not clear where it happened.